.obs/workflows.yml

```
@@ -0,0 +1,34 @@
+---
+pr:
+  steps:
+    - branch_package:
+        source_project: devel:openQA
+        source_package: os-autoinst-distri-opensuse-deps
```

I think that should be:
```suggestion
        source_package: os-autoinst-scripts
```
https://build.opensuse.org/package/show/devel:openQA/os-autoinst-scripts

---

@perlpunk I am not sure if this is the only thing I need. before we merge I guess we have to add the tokens. and what else?